### ISSUE TITLE
possible conflict with libtvm

### ISSUE THREAD

**Issue by ngam** [conda-forge/pytorch-cpu-feedstock#94]
Hi @conda-forge/pytorch-cpu. Not sure if anyone wanted to cross pytorch with libtvm, but I am currently working on fixing [latter](https://github.com/conda-forge/libtvm-feedstock/pull/8) and getting it up to speed (to be used with mxnet, etc.) and they have [this section](https://tvm.apache.org/docs/install/from_source.html) upstream:

> If you are a PyTorch user, it is recommended to set `(USE_LLVM "/path/to/llvm-config --link-static")` and `set(HIDE_PRIVATE_SYMBOLS ON)` to avoid potential symbol conflicts between different versions LLVM used by TVM and PyTorch.

Not sure if you have any preference on how this should be addressed. Let me know.

For now, I am simply adding llvm support; I can see us making a special edition of libtvm to be used alongside pytorch and setting a conflict otherwise.

**Comment by hmaarrfk** [comment]
You raise many good issues, them they get closed without explanation.

Do you want to share why this one was closed?

**Comment by ngam** [comment]
> You raise many good issues, them they get closed without explanation.
> 
> Do you want to share why this one was closed?

Hey @hmaarrfk! Sorry I just saw your comment --- I have muted all my notifications. As a result, if something isn't open, it simply wouldn't cross my attention. I came to this feedstock to check where we are in the general pytorch push.

Okay, yes, sorry — this is like my new-year resolution to close issues if they don't get attention within a few weeks; if something is important to revisit, it is better to just open a fresh issue with a fresh perspective. Especially with open-source endeavors, I find this to be the most productive way. I know it is weird, and may prove disruptive, but hopefully someone can reopen an issue if they think I made a mistake. I tend to think if something is important, it will resurface (at least in my head). Maybe I should stop this bad habit? 

But anyway, the direct reason why I closed this for now is actually simple: I couldn't figure out a way to compile mxnet with libtvm, and so for now, I am pausing my pursuit of tinkering with tvm until at least a few more months. I do want to revisit this, but I thought I would just open a fresh issue then and see what we can do. I also was thinking of looking more into how a tvm-based pytorch compilation would work before throwing it out here. Remember when I was trying to get pytorch to work with Apple's Accelerate and then realized it was not worth it? I don't want to waste everyone's time with speculative things :) 

Btw, a few weeks ago I had HPC people at an institution I am affiliated with add the tensorflow cuda override documentation as you suggested a while back!
